src/styles/GlobalStyles.tsx

```diff
@@ -0,0 +1,25 @@
+import {StyleSheet} from 'react-native';
+
+const GlobalStyles = StyleSheet.create({
```

Comment:
yeah, I understand. I wanted these styles outside of the component, as I feel, this could be used globally. So maybe this is part of theming as well, same as colors, like you mentioned in the other comment.
For now, I will pull it into the component, but I guess, we should talk about the global styles

Reply:
Yes, what you're saying is definitely a thing. Part of my work will be adding a reusable View/Text which preconfigured options. That's the only way I've seen to manage things consistently in RN